tests/channels/test_d0_to_ks_kp_km.py

```diff
@@ -3,7 +3,7 @@
 """
 
 import logging
-import dot_parser
+import dot_parser  # type: ignore
```

Comment:
Actually this mypy check can be fixed for the whole framework in the `setup.cfg` file, like this:
https://github.com/ComPWA/expertsystem/blob/02265cfe1b3a06a42a3a3405c8c42dc546be4eb4/setup.cfg#L99-L100
(note the alphabetical order, as in https://github.com/ComPWA/expertsystem/pull/216#discussion_r471613796)